index.bs

```diff
@@ -72,32 +66,15 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The Gyroscope's associated <a>Sensor subclass</a>
+The Gyroscope's associated {{Sensor}} subclass
```

Comment:
Same capitalization issue here (and throughout the spec). Would be good to link these, too. Overall, it seems the model on the generic sensor side isn't clear enough. Need to work on it.